src/Symfony/Component/Validator/Mapping/Loader/XmlFileLoader.php

```diff
@@ -182,12 +184,23 @@ protected function parseFile($path)
         return simplexml_import_dom($dom);
     }
 
-    /**
-     * Loads the validation metadata from the given XML class description.
-     *
-     * @param ClassMetadata $metadata         The metadata to load
-     * @param array         $classDescription The XML class description
-     */
+    private function loadClassesFromXml()
+    {
+        // This method may throw an exception. Do not modify the class'
```

Comment:
@Ener-Getick looks like it is, [see below](https://github.com/symfony/symfony/pull/19485/files#diff-62bcd5bcf87578228004c4b2e075db3cR193)